Makefile

```diff
@@ -122,7 +122,7 @@ pluginsandtagboxes:
 
 all: install
 
-install: slash pluginsandtagboxes
+install: stamp/append-apache-config slash pluginsandtagboxes
```

Comment:
Append Apache config only on install, or you'll wind up with 2 appends. Once during make, once during make install.